commands/birthday.py

```diff
@@ -0,0 +1,63 @@
+import bot
+import discord
+import json
+import datetime
+
+
+def import_command():
+
+    birthday_commands = discord.app_commands.Group(
+        name='birthday',
+        description='Get birthday celebrations from frank'
+    )
+
+    # Add Command group to tree
+    bot.tree.add_command(birthday_commands)
+
+    # Command Info
+    @birthday_commands.command(
+        name="set",
+        description='Get birthday celebrations from frank'
+    )
+    # Code to Run Here
+    async def self(interaction: discord.Interaction, mm: int, dd: int, yyyy: int = 1):
+        timestamp = datetime.datetime(yyyy, mm, dd).timestamp()
+        birthdays = json.loads(open("birthdays.json", "r").read())
+        birthdays[interaction.user.id] = timestamp
+        update = open("birthdays.json", "w")
+        update.write(json.dumps(birthdays, indent=3))
+        update.close()
```

Comment:
Use a `with` statement to avoid the need to manually `update.close()` the file

```suggestion
        with open("birthdays.json", "w") as update:
            update.write(json.dumps(birthdays, indent=3))
```

Reply:
yeah i tried that but had issues, but if that will work than thats what ill do